_data/try.yml

```diff
@@ -0,0 +1,18 @@
+- title: Try Jupyter with Python
+  description: |
+    A tutorial introducing basic features of Jupyter notebooks
```

Comment:
Since we are going to have the JupyterLab demo here shortly, perhaps adding classic to the description to something like:
A tutorial introducing the basic features of the classic Jupyter notebook and the IPython kernel.

Reply:
I agree